README.md

```diff
@@ -4123,6 +4159,59 @@ Message Type: **response**
 |`resultCode`|Result|True|See Result|
 
 
+### SetCloudAppProperties
+Message Type: **request**
+
+RPC used to enable/disable a cloud application and set authentication data
```

Comment:
The description should also mention the other parameters which can be set (like the cloud_transport_type, hybrid_app_preference, etc.). From the current description it would seem that `SetCloudAppProperties` can only be used to enable/disable the cloud app and/or set the auth_token